mkdocs.yml

```diff
@@ -22,32 +24,49 @@ markdown_extensions:
   - toc:
       permalink: true
 nav:
-  - Introduction: index.md
-  - Concepts:
-      API overview: concepts/api-overview.md
-      Security Model: concepts/security-model.md
-      Implementation Guidelines: concepts/guidelines.md
-  - API Types:
-      GatewayClass: api-types/gatewayclass.md
-      Gateway: api-types/gateway.md
-      HTTPRoute: api-types/httproute.md
-  - Guides:
-    - Getting started: guides/getting-started.md
-    - Simple Gateway: guides/simple-gateway.md
-    - HTTP routing: guides/http-routing.md
-    - HTTP traffic splitting: guides/traffic-splitting.md
-    - Multiple namespaces and routes: guides/multiple-ns.md
-    - TLS: guides/tls.md
-    - TCP routing: guides/tcp.md
-  - References:
-    - API specification: references/spec.md
-    - Releases: references/releases.md
-    - Implementations: references/implementations.md
-  - ... | geps/gep-*.md
+  - Overview:
+    - Introduction: index.md
+    - Concepts:
+        API overview: concepts/api-overview.md
+        Security Model: concepts/security-model.md
+        Implementation Guidelines: concepts/guidelines.md
+    - Implementations: implementations.md
+  - Reference:
+    - v1alpha2(unreleased):
+      - API Types:
+          GatewayClass: v1alpha2/api-types/gatewayclass.md
+          Gateway: v1alpha2/api-types/gateway.md
+          HTTPRoute: v1alpha2/api-types/httproute.md
+      - Guides:
+        - Getting started: v1alpha2/guides/getting-started.md
+        - Simple Gateway: v1alpha2/guides/simple-gateway.md
+        - HTTP routing: v1alpha2/guides/http-routing.md
+        - HTTP traffic splitting: v1alpha2/guides/traffic-splitting.md
+        - Multiple namespaces and routes: v1alpha2/guides/multiple-ns.md
```

Comment:
Most of these are out of date as captured by #783, until we update them, do you mind adding a warning on those pages similar to what I had in #778? `!!! danger "This page has not been updated for v1alpha2 yet."`